docs/publish/host-an-office-add-in-on-microsoft-azure.md

```diff
@@ -1,225 +1,180 @@
-    In the browser address bar, change the URL for the website so that it uses HTTPS and press  **Enter** to confirm that the HTTPS protocol is enabled. The Office Add-in model requires add-ins to use the HTTPS protocol.
-    
-6. On the  **web apps** page, choose the new website.
-    
-7. Under  **Publish your app**, choose  **Download the publish profile**, which saves the publish profile to your computer. Remember the file name and the location because you will need this later.
-    
-    The publish profile contains your credentials and enables you to securely publish to Azure. 
+      - Choose **OS** for your site.
     
+      - Choose the **App Service plan** to use for creating this site. If you create a new plan, you also need to name it.
+       
+      - Choose **Create**.
 
-## Step 4: Create an Office Add-in in Visual Studio
+4. Choose **Notifications** (the bell icon that is located along the top edge of the Azure portal) and then choose the **Deployments succeeded** notification to open the site's **Overview** page in the Azure portal.
 
+    >**Note:** The notification will change from **Deployment in progress** to **Deployments succeeded** when the site deployment completes.
 
+5. In the **Essentials** section of the site's **Overview** page in the Azure portal, choose the URL that is displayed under **URL**. Your browser opens and displays a webpage with the message "Your App Service app has been created." 
+    
+6. In the browser address bar, change the URL for the web app so that it uses HTTPS and press **Enter** to confirm that the HTTPS protocol is enabled. The Office Add-in model requires add-ins to use the HTTPS protocol.    
+
+## Step 4: Create an Office Add-in in Visual Studio
 
 1. Start Visual Studio as an administrator.
     
-2. Choose  **File** > **New** > **Project**.
-    
-3. Under  **Templates**, expand  **Visual C#** (or **Visual Basic**), expand  **Office/SharePoint**, and then choose  **Office Add-ins**.
-    
-4. Choose  **Office Add-in**, and then choose **OK** to accept the default settings.
+2. Choose **File** > **New** > **Project**.
     
-5. When  **Create Office Add-in** appears, leave the default choice for a Task pane add-in and choose **Next**.
+3. Under **Templates**, expand **Visual C#** (or **Visual Basic**), expand **Office/SharePoint**, and then choose **Add-ins**.
     
-6. On the next page, clear all check boxes except for Word, then choose  **Finish**.
-    
-Your basic Office Add-in is created and ready to publish to Azure.
-Because we're focusing on how to publish to Azure, you won't make any changes to the sample add-in that you created by using the standard Office Add-in template in Visual Studio.
-
-## Step 5: Publish your Office Add-in to the Azure website
+4. Choose **Word Web Add-in**, and then choose **OK** to accept the default settings.
+       
+Visual Studio creates a basic Word Web Add-in that you'll be able to publish as-is, without making any changes to its web project.
```

Comment:
I recommend that we just call it a "Word add-in", not a "Word Web Add-in" (we don't need the caps).